webapp/src/shared/components/Row/Row.css

```diff
@@ -2,7 +2,6 @@
   display: flex;
   width: 100%;
   min-width: fit-content;
-  height: 4.5rem;
   padding: 0.9rem 1.56rem;
   border-radius: 0.44rem;
```

Comment:
praise: Looks good. Thank you for removing the height.

suggestion: I think these rem values are using 20px as a base, and for the mobile version, the base is 16px
```
  padding: 1rem 1.75rem;
  border-radius: 0.5rem;
  column-gap: 1.25rem;
```
The `column-gap` in Figma was 22px. I think, for consistency, we could use a multiple of 4. We could change it to 20px